nixos/modules/services/databases/postgresql.nix

```
@@ -597,12 +603,26 @@ in
                   clauseSqlStatements = attrValues (mapAttrs (n: v: if v then n else "no${n}") filteredClauses);
 
                   userClauses = ''$PSQL -tAc 'ALTER ROLE "${user.name}" ${concatStringsSep " " clauseSqlStatements}' '';
+
+                  ensureUserPasswords = lib.optionalString
+                    (user.ensurePasswordFile != null)
+                    ''
+                      $PSQL -tA <<'EOF'
```

```suggestion
                  ensureUserPasswords = lib.optionalString (user.ensurePasswordFile != null) ''
                    $PSQL -tA <<'EOF'
```
we can keep the indentation a bit lower here